doc/requirements.txt

```
@@ -21,7 +22,7 @@ sphinxcontrib-devhelp==1.0.2
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-htmlhelp==2.0.1
 sphinxcontrib-serializinghtml==1.1.5
-tensorflow==2.11.1; platform_machine == "x86_64"
+tensorflow==2.19.0; platform_machine == "x86_64"
```

```suggestion
tensorflow==2.18.0; platform_machine == "x86_64"
```
`2.19` is not necessary. Let's pin consistent with other places